File: fix.dart
import 'dart:io';

void main() {
  final file = File('mobile/lib/src/features/reminder/presentation/screens/medication_reminders_screen.dart');
  String text = file.readAsStringSync();
  
  text = text.replaceAll(
    '\'\\\\ tane aktif hatÄ±rlatÄ±cÄ±nÄ±z var.\'',
    "'\\\ tane aktif hatırlatıcınız var.'"
  );

  file.writeAsStringSync(text);
}

File: fix.py
import re

with open('mobile/lib/src/features/reminder/presentation/screens/medication_reminders_screen.dart', 'r', encoding='utf-8') as f:
    text = f.read()

text = re.sub(r\"'\\s*aktif hat[\\s\\S]*?var\.'\", r\"'\ tane aktif hatırlatıcınız var.'\", text)

with open('mobile/lib/src/features/reminder/presentation/screens/medication_reminders_screen.dart', 'w', encoding='utf-8') as f:
    f.write(text)
